app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -182,3 +182,48 @@ table
       padding: 4px
       font-size: 0.8em
       color: #505050
+
+// tooltips with html support
+span.op_tooltip
+  visibility: hidden
+  opacity: 0
+  transition: visibility 0s linear 0.2s, opacity 0.2s linear
+  position: absolute
+  border: 2px solid #a7cbe1
+  background: #e3f5ff
+  padding: 15px
+  font-style: italic
+  line-height: 18px
+  width: 353px
```

Comment:
@floriank I suppose the values somehow were got from [here](http://clients.zzmedia.net/openproject/my-account/change-password/)